userscript/lovely-forks.user.js

```diff
@@ -0,0 +1,501 @@
+// ==UserScript==
+// @name         Lovely Forks
+// @namespace    musically-ut
+// @version      2.6.2
+// @description  Show notable forks of GitHub projects.
+// @homepage     https://github.com/musically-ut/lovely-forks
+// @icon         https://github.com/musically-ut/lovely-forks/raw/master/userscript/icon.png
```

Comment:
👍 

Alright, I'll leave the separate logo here and will refactor it after all the merging is done in one go.